master-keyids

```diff
@@ -1 +0,0 @@
-1F35A37F9D85CBC2B74F595E37CE6162C969C50E joske
```

Comment:
Both files `master-keyids` and `master/*.asc` are currently not used by the makefiles. Also it seems the `update-keys` was not really used(?)

Reply:
I think the script was wip and never finished